Website Files/svtechsol.css

```diff
@@ -745,6 +745,32 @@ body {
 	font-size: 10px;
 }
 
+@media only screen and (max-width: 1330px) {
+	.banner-description {
```

Comment:
looks like your tabs are too big...
normally they should be 4 spaces or 2 spaces